python/wtte/transforms.py

```diff
@@ -2,15 +2,19 @@
 from __future__ import division
 from __future__ import print_function
 
+import sys
```

Comment:
Importing sys is considered bad practice (flake8). I fixed the issue of basestring with commit 7034b9949c5718dbf25bfa8d669d7f4bd2be2af6 by surpassing it